_pages/icml_2020/icml-2020-presenters.html

```diff
@@ -0,0 +1,10 @@
+---
+layout: main
+permalink: /icml-2020-presenters
+title: LXAI @ ICML - Presenters
+published: true
```

Comment:
Until presenters are chosen and information in the includes files are updated, published should be false.